src/tool/cppwinrt/strings/base_fast_forward.h

```diff
@@ -1,3 +1,13 @@
+#include <cstddef>
+#include <atomic>
```

Comment:
is std::atomic safe for non-throwing code?

Reply:
Yes, it just compiles down to the various intrinsics and is explicitly noexcept.

Reply:
ideally would be good to add a test case for this from the repo validating usage of just this header with /Eh-